src/style/main.less

```diff
@@ -281,3 +151,32 @@ input {
   html { font-size: @base-11; }
 
 }
+
+/* Setup Color Stops */
+.light {
```

Comment:
I think all these class names need to be prefixed. Something like `.color-stop__light`, `color-stop__original`, etc.